include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -2137,16 +2143,20 @@ ::FEEvaluationBase (const FEEvaluationBase<dim,n_components_,Number> &other)
   mapping_info       (other.mapping_info),
   stored_shape_info  (other.stored_shape_info),
   data               (other.data),
-  cartesian_data     (other.cartesian_data),
-  jacobian           (other.jacobian),
-  J_value            (other.J_value),
-  quadrature_weights (other.quadrature_weights),
-  quadrature_points  (other.quadrature_points),
-  jacobian_grad      (other.jacobian_grad),
-  jacobian_grad_upper(other.jacobian_grad_upper),
-  cell               (other.cell),
-  cell_type          (other.cell_type),
-  cell_data_number   (other.cell_data_number),
+  cartesian_data     (0),
```

Comment:
This is related to the initialization check I added in `quadrature_point`: When you copy an object, you don't want it to be initialized to the state of the object copied from. In case the user unintentionally forgets to call `reinit()`, there will be some weird state and some pointers would be correctly set whereas others are invalid. I'd prefer to have invalid pointers so it gets noticed.